### PR TITLE
[14.0][IMP]l10n_ch_invoice_reports: remove isr

### PR DESCRIPTION
**Pull request by TDu**
This is based on

* https://github.com/OCA/l10n-switzerland/pull/672

But ISR being not used anymore, the related parts have been removed.

Also with the help of this PR (still need to be merged)

* https://github.com/odoo/odoo/pull/79318

The code could be simplified.

### CODE REVIEW

**Comment by simahawk** [comment]
What's the status of https://github.com/OCA/l10n-switzerland/pull/672 ?
CC @ajaniszewska-dev

**Comment by ajaniszewska-dev** [comment]
> What's the status of #672 ? CC @ajaniszewska-dev

@simahawk  i am not fully sure #672 was working properly. and afair it is not used anywhere - i just wanted to introduce setting to V14. If this PR is based on that one and removes ISR + works well, i guess 672 may be closed.

File: l10n_ch_invoice_reports/data/reports.xml [hunk 1]
```diff
@@ -1,49 +1,15 @@
...
-        <field name="attachment_use" eval="False" />
-        <field name="attachment" />
-        <field name="binding_model_id" ref="account.model_account_move" />
-        <field name="binding_type">report</field>
-    </record>
+      <report
+        id="account_move_payment_report"
+        model="account.move"
+        report_type="qweb-pdf"
+        string="Invoice with payment slip(s)"
```
Comment (ajaniszewska-dev) [inline l10n_ch_invoice_reports/data/reports.xml:7]:
```suggestion
        string="Invoice with payment slip"
```

as we don't have big choice of payslips now

**Review by ajaniszewska-dev — APPROVED** [review]
LG, minor comment.

**Review by simahawk — APPROVED** [review]
LG

**Pull request merged**